pkg/util/validation/limits.go

```diff
@@ -106,6 +111,8 @@ func (l *Limits) RegisterFlags(f *flag.FlagSet) {
 	f.IntVar(&l.CardinalityLimit, "store.cardinality-limit", 1e5, "Cardinality limit for index queries. This limit is ignored when running the Cortex blocks storage. 0 to disable.")
 	f.DurationVar(&l.MaxCacheFreshness, "frontend.max-cache-freshness", 1*time.Minute, "Most recent allowed cacheable result per-tenant, to prevent caching very recent results that might still be in flux.")
 
+	f.DurationVar(&l.EvaluationDelay, "ruler.evaluation-delay-duration", 0, "Duration to delay the evaluation of rules to ensure they underlying metrics have been pushed to cortex.")
```

Comment:
```suggestion
	f.DurationVar(&l.EvaluationDelay, "ruler.evaluation-delay-duration", 1*time.Minute, "Duration to delay the evaluation of rules to ensure the underlying metrics have been pushed to cortex.")
```

Reply:
A typo and if we're making this per tenant I reckon `60s` might be "safer" approach?

Reply:
Fixed typo, but we don't change default flag values for backwards compatibility reasons.  Don't want to surprise people on upgrades.